src/backend/src/prisma/seed-data/teams.seed.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of FinishLine by NER and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+const about = `# About the Project
```

Comment:
This is for the whole about part/ the way descriptions are being stored rn. I think if we wanna add all these options for variation in text size, indentation, etc. inside of the description box, we should do it in a separate ticket and make it it's own component in it's own file. Because imo, the descriptions are not super readable right now and don't look very clean. Additionally they don't have any standardization rn. Even if it's just for the inital demo for the teams page, I'd prefer we just make it some simple text and add in the complexity within the description later.

Reply:
are you saying just take out the markdown aspect of it?

Reply:
@anthonybernardi can you explain what markdown is/ why we've chosen to use it here? I can't find details on when it was added to this ticket in the slack. I'm not against using it necessarily, just confused.

Reply:
true, definitely need to get the spec doc out